### PR TITLE
Release notes on releases

### PR DESCRIPTION
**Pull request by hannesa2**
close https://github.com/gitx/gitx/issues/287

### CODE REVIEW

File: .github/changelog-configuration.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "gradle-wrapper"
   ],
   "sort": "ASC",
+  "tag_resolver.filter": "^\\d",
```
Comment (claytonrcarter) [inline .github/changelog-configuration.json:29]: Sorry I wasn't more clear (their docs aren't clear either) but I think this needs to be a JSON object: `"tag_resolver": { "filter": "^\\d" },`

Comment (hannesa2) [reply .github/changelog-configuration.json]: I'm not sure if it needs the `pattern`.

https://github.com/mikepenz/release-changelog-builder-action/commit/ed954db1f67d8c6800eaee966fd43dba85097fac#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L206-L208

Comment (claytonrcarter) [reply .github/changelog-configuration.json]: Oh! Yes! Good catch! It looks like their example is up-to-date:
Screen Shot 2022-02-17 at 8 15 17 AM

but their config docs are out of date:
Screen Shot 2022-02-17 at 8 15 08 AM

Anyway, yes, I think that 
```json
"tag_resolver": {
  "filter": {
    "pattern": "^\\d"
  }
}
```
is the way to go.

**Comment by hannesa2** [comment]
I tested it with a release on my fork https://github.com/hannesa2/gitx/runs/5234832080?check_suite_focus=true

image

**Comment by hannesa2** [comment]
Unfortunately, this didn't solved the issue

**Comment by claytonrcarter** [comment]
Why do you say it's not working? The 0.19 update window just popped up for me and it only includes the 4 PRs merged since we released 0.18.1:

Screen Shot 2022-03-07 at 4 55 03 PM

Did you see something else, or expect something else?

**Comment by hannesa2** [comment]
Because I manually deleted the other entries in release page

**Comment by hannesa2** [comment]
Please see the automatic release mail (It's just a part of it)
image

**Comment by hannesa2** [comment]
I guess this is the reason 
image

**Comment by hannesa2** [comment]
I reverted this changes and it looks promising https://github.com/gitx/gitx/pull/300#issuecomment-1061433362

**Pull request merged**